src/hotspot/cpu/aarch64/aarch64.ad

```diff
@@ -11143,6 +11143,22 @@ instruct mulHiL_rReg(iRegLNoSp dst, iRegL src1, iRegL src2, rFlagsReg cr)
   ins_pipe(lmul_reg_reg);
 %}
 
+instruct umulHiL_rReg(iRegLNoSp dst, iRegL src1, iRegL src2, rFlagsReg cr)
```

Comment:
`rFlagsReg cr` is not used.

Reply:
Yes, I noticed, but since there are many more unused `rFlagsReg cr` in the file and since I don't know its semantics, I prefer to just leave it as it is.